src/software/SfM/main_ComputeCCTagStructureFromKnownPoses.cpp

```diff
@@ -19,6 +19,319 @@ using namespace openMVG;
 using namespace openMVG::sfm;
 namespace bfs = boost::filesystem;
 
+ /**
+   * Apply a similarity transformation to the reconstruction so that the centroid
+   * of the optical centers is the origin of the world coordinate system, a 
+   * dominant plane P is fitted to the set of the optical centers and the scene 
+   * aligned so that P roughly define the (x,y) plane, and the scale is set so 
+   * that the optical centers RMS is of sqrt(2).
+   * (Hartley-like normalization, p.180)
+   *
+   * \param[out] sfmData   Transformed reconstruction.
+   */
+void conditionSfMFromFromCameras(
```

Comment:
maybe we can move those functions inside the lib as they could be useful for other app